website/versioned_docs/version-0.2.x/guides/creating-composites/overview.mdx

```diff
@@ -24,13 +24,21 @@ After a schema file has been created, it must be converted to a composite to be
   ]}>
   <TabItem value="api">
 
+Make sure you have the following packages installed, before running this code:
+ * `@ceramicnetwork/http-client`
+ * `@composedb/devtools-node`
+
+ In order to create a Composite using the API, you also need to have an authenticated DID instance. You can use the `did-session` package from `js-did` to achieve this: [DIDSession.DID Documentation](https://did.js.org/docs/api/classes/did_session.DIDSession#did)
```

Comment:
That's incorrect, you can't create composites with `did:pkh`, it has to be `did:key`